src/adapter/src/rbac.rs

```diff
@@ -508,17 +551,11 @@ fn generate_required_privileges(
             connection: _,
             validate: _,
         }) => {
-            let mut privileges = vec![(
+            vec![(
                 SystemObjectId::Object(name.qualifiers.clone().into()),
                 AclMode::CREATE,
                 role_id,
-            )];
-            privileges.extend(generate_item_usage_privileges(
```

Comment:
These can all get removed because it now occurs in check_item_privilege?

Reply:
Yep.